docs/source/interactive/reference.rst

```diff
@@ -609,10 +609,76 @@ python, see :ref:`configure_start_ipython`.
 
 It is also possible to embed an IPython shell in a namespace in your Python code.
 This allows you to evaluate dynamically the state of your code,
-operate with your variables, analyze them, etc. Note however that
-any changes you make to values while in the shell do not propagate back
-to the running code, so it is safe to modify your values because you
-won't break your code in bizarre ways by doing so.
+operate with your variables, analyze them, etc. For example, if you run the 
+following code snippet:
+
+``` python
```

Comment:
I don't think this syntax works in restructuredtext. To make a code block, use a double colon and indent it:

```rst
following code snippet::

     import IPython

     a = 42
     IPython.embed()
```

(If you've already tried building the docs and the markdown-style code blocks work, that's fine, of course)